website/members/models/member.py

```diff
@@ -196,3 +196,13 @@ def tpay_enabled(self):
             and bank_accounts.exists()
             and any(x.valid for x in bank_accounts)
         )
+
+    @property
+    def tpay_balance(self):
+        """The total amount of open, unprocessed Thalia Pay payments"""
+        payments = Payment.objects.filter(
+            Q(paid_by=self, type=Payment.TPAY)
+            & (Q(batch__isnull=True) | Q(batch__processed=False))
+        )
+        total = payments.aggregate(Sum("amount"))["amount__sum"]
+        return -1 * total if total else 0
```

Comment:
When would total be None or False? Isn't a sum of nothing 0?

Reply:
Nope apparently the sum of nothing is `None`